git-archive-all

```diff
@@ -128,7 +163,9 @@ class GitArchiver(object):
                 add_file(file_path, arcname)
         else:
             archive = None
-            archiver = lambda file_path, arcname: self.LOG.info("{} => {}".format(file_path, arcname))
+
+            def archiver(file_path, arcname):
+                self.LOG.info("{0} => {1}".format(file_path, arcname))
```

Comment:
Ah sorry, misunderstood this one.
This is about another pep8 requirement : 
`Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier.`